.github/workflows/ci.yml

```
@@ -57,12 +33,82 @@ jobs:
           --health-timeout 5s
           --health-retries 10
     steps:
-    - uses: actions/checkout@v3.5.0
-    - name: SBT
-      uses: ./.github/actions/sbt
-      env:
-        USE_DOCKER_FOR_TESTS: false
-        ES6_TEST_URL: http://elasticsearch:9200
-        LOCALSTACK_ENDPOINT: http://localstack:4566
-      with:
-        args: clean compile test:compile test
+      - uses: actions/checkout@v4
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+      - name: Kahuna (client-side)
+        working-directory: ./kahuna
+        run: |
+          npm install-clean
+          npm run undist
+          npm test
+          npm run dist
+      - name: install image libs needed for tests
+        run: |
+          sudo apt-get -y update
+          sudo apt-get -y install graphicsmagick
+          sudo apt-get -y install graphicsmagick-imagemagick-compat
+          sudo apt-get -y install exiftool
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'corretto'
```

```suggestion
          distribution: 'corretto'
          cache: 'sbt'
```
to make sure GHA caches downloaded libs (likely to be important for performance)